http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/GceInterpreter.scala

```diff
@@ -370,6 +370,9 @@ class GceInterpreter[F[_]](
         metadataToAdd = metadata,
         metadataToRemove = Set("startup-script-url")
       )
+      // This sleep is added here because as of 2/1/2022, we started seeing increased amount of `RESOURCE_NOT_READY` error.
+      // Google support says this is due to the setMetadata API and start VM API is called too close to each other.
+      _ <- F.sleep(config.gceConfig.waitBeforeStartVM)
       _ <- googleComputeService.startInstance(googleProject,
```

Comment:
Should this also (or instead) have some kind of retry / backoff for the RESOURCE_NOT_READY error?

Reply:
`RESOURCE_NOT_READY` happens without an error to client. The API response will be sth like success, and operation pending. A few min later, on GCP side, `RESOURCE_NOT_READY` will happen, but still when leo pings GCP about "what's the status of this VM", GCP will just respond with the VM status being `TERMINATED`

Reply:
Ah, I see - thanks.